src/app/components/PlayersBoard.ts

```diff
@@ -64,12 +69,50 @@ export class PlayersBoard {
       playersDataPanel,
       playerName,
       playerAvatar,
+      playerTimer,
     );
     Render.childrenInjector(
       playersBoardContainer,
       playersDataPanel,
       herdView,
     );
+    /*
+    this.view.stopTimer();
+    setTimeout(() => this.hideTimer(), 10);
+  }
+
+  private hideTimer(): void {
+    (document.querySelector(
+      '#time-left',
+    ) as HTMLElement).style.display = 'none';
+  }
+
+  /**
+   * Updates timer on player panel
+   * @param timeLeft accepts number value for time left
+   */
+    /*updateTime(timeLeft: number): void {
+    const timer = document.querySelector('#time-left') as HTMLElement;
```

Comment:
Also, target the right timer. Or, if the timer is deleted after every round (need to think of a way to remove it), it shold be inserted at the beginning of the round and removed at the end. Then we would have just one element of `id:time-left` and that would be ok.